src/main/scala/org/constellation/consensus/EdgeProcessor.scala

```diff
@@ -97,6 +97,8 @@ object EdgeProcessor extends StrictLogging {
         data
       )
 
+      dao.recentBlockTracker.put(data)
```

Comment:
should this be moved into cb.store?

Reply:
Yeah I can do that, it was removed from the other merge but I was just reverting this.